felix/calc/policy_resolver.go

```diff
@@ -111,15 +111,13 @@ func (pr *PolicyResolver) OnUpdate(update api.Update) (filterOut bool) {
 			pr.markEndpointsMatchingPolicyDirty(key)
 		}
 	}
-	pr.maybeFlush()
```

Comment:
IIUC, the policyresolver now only flushes when the calcgraph does - are there any types of updates which should still be flushed more aggressively / decided explicitly by the pol-resolver / how much risk is there of the CG's flush strategy having a blind spot? More of a question than feedback :smile:

Reply:
It's also the case that the policy resolver is at the bottom of the calc graph already so it's the last thing in its call path that emits events (so there's no chance of reordering events here, I don't think)